### PR TITLE
Fix broken links

### PR DESCRIPTION
**Pull request by letanure**
Found some broken links on the docs, while I was reading, for each broken one, committed an updated version

**Approach and changes**
- image with relative link to current repo and branch
- link fixed to the right page
- replaced some old pages references for a github package reference
- removed one section about a static css version, not found

## Definition of done

* [ ] Development completed
* [ ] Reviewers assigned
* [ ] Unit and integration tests
* [ ] Meets minimum browser support
* [ ] Meets accessibility requirements

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ We want to foster an inclusive and friendly community around our Open Source eff
 
 If you feel another member of the community violated our CoC or you are experiencing problems participating in our community because of another individual's behavior, please get in touch with our [maintainers](README.md#maintainers). We will enforce the CoC.
 
-![Overview diagram of the contribution model](https://github.com/sumup-oss/circuit-ui/tree/main/assets/contribution-model.jpg?raw=true)
+![Overview diagram of the contribution model](/assets/contribution-model.jpg?raw=true "Overview diagram of the contribution model")
```
Comment (connor-baer) [inline CONTRIBUTING.md:9]: Thanks for adding an alternative description 🙌🏻

The relative link would break the image in Storybok (this file is turned into the [Contributing → Overview](https://circuit.sumup.com/?path=/story/introduction-contributing-overview--page) page. 

Let's fix the raw URL instead. 
```suggestion
![Overview diagram of the contribution model](https://github.com/sumup-oss/circuit-ui/raw/main/assets/contribution-model.jpg "Overview diagram of the contribution model")
```

File: docs/features/icons.stories.mdx [hunk 2]
```diff
@@ -6,7 +6,7 @@ import { Meta, Intro, Icons } from '../../.storybook/components';
 
 <Intro>
 
-The icons used throughout Circuit UI come from SumUp's icon library, [`@sumup/icons`](Packages/icons), which is a required peer dependency of `@sumup/circuit-ui`. Refer to its [documentation](Packages/icons) for usage instructions.
+The icons used throughout Circuit UI come from SumUp's icon library, [`@sumup/icons`](https://github.com/sumup-oss/circuit-ui/tree/main/packages/icons), which is a required peer dependency of `@sumup/circuit-ui`. Refer to its documentation for usage instructions.
```
Comment (connor-baer) [inline docs/features/icons.stories.mdx:9]: This link works, try it [here](https://circuit.sumup.com/?path=/story/features-icons--page). The Storybook pages for the packages are generated from the package READMEs, so to view them locally, you need to run `yarn copy-docs` first. 

Could you revert this change and add the `yarn copy-docs` command to the `yarn bootstrap` script in the `package.json` file, please?

File: docs/features/theme.stories.mdx [hunk 3]
```diff
@@ -18,7 +18,7 @@ import { Heading, SubHeading, Text } from '../../packages/circuit-ui';
 
 <Intro>
 
-The theme used throughout Circuit UI comes from SumUp's design token library, [`@sumup/design-tokens`](Packages/design-tokens), which is a required peer dependency of `@sumup/circuit-ui`. Refer to its [documentation](Packages/design-tokens) for usage instructions.
+The theme used throughout Circuit UI comes from SumUp's design token library, [`@sumup/design-tokens`](https://github.com/sumup-oss/circuit-ui/tree/main/packages/design-tokens), which is a required peer dependency of `@sumup/circuit-ui`. Refer to its documentation for usage instructions.
```
Comment (connor-baer) [inline docs/features/theme.stories.mdx:21]: Same as above.

File: docs/introduction/welcome.stories.mdx [hunk 4]
```diff
@@ -37,7 +37,7 @@ Set up a new app with Circuit UI or add it to an existing project.
 
 File a bug report, suggest a change, or open a pull request.
 
-[Contribute](Introduction/Contributing/Overview)
+[Contribute](https://github.com/sumup-oss/circuit-ui/blob/main/CONTRIBUTING.md)
```
Comment (connor-baer) [inline docs/introduction/welcome.stories.mdx:40]: Same as above. This page is generated by the `yarn copy-docs` script.

**Review by connor-baer — CHANGES_REQUESTED** [review]
Thanks for contributing these fixes! ✨

File: docs/introduction/welcome.stories.mdx (hunk 4 above)
Comment (letanure) [reply docs/introduction/welcome.stories.mdx:40]: done

File: docs/features/theme.stories.mdx (hunk 3 above)
Comment (letanure) [reply docs/features/theme.stories.mdx:21]: done

File: docs/introduction/welcome.stories.mdx (hunk 4 above)
Comment (letanure) [reply docs/introduction/welcome.stories.mdx:40]: checking the live version, I found those errors, with the following URL, notice the `undefined`

https://circuit.sumup.com/iframe.html?id=features-theme-undefined--page&viewMode=story

File: docs/features/theme.stories.mdx (hunk 3 above)
Comment (connor-baer) [reply docs/features/theme.stories.mdx:21]: I still see the change 🤔

File: docs/introduction/welcome.stories.mdx (hunk 4 above)
Comment (connor-baer) [reply docs/introduction/welcome.stories.mdx:40]: Where did you find this URL?

Comment (connor-baer) [reply docs/introduction/welcome.stories.mdx:40]: Found it on [this page](https://circuit.sumup.com/?path=/story/introduction-getting-started--page). Would you mind fixing that, please?

File: docs/features/icons.stories.mdx (hunk 2 above)
Comment (letanure) [reply docs/features/icons.stories.mdx:9]: added, but got some error on eslint on package.json, added on eslintignore

File: docs/features/theme.stories.mdx (hunk 3 above)
Comment (letanure) [reply docs/features/theme.stories.mdx]: forgot that was my remote for push was in my fork

**Review by connor-baer — APPROVED** [review]

**Comment by connor-baer** [comment]
Fixed in cb343cd and 7b83f27.
